### DISCUSSION TITLE
How to connect to lavalink using latest docker image

### DISCUSSION THREAD

**Discussion by BONEE4** [lavalink-devs/Lavalink#817] [Q&A] [answered]
## Description
I wanted to know how do I connect to lavalink using docker. I tried using localhost:2333 and it doesn't work I also tried using docker ips

## Version info
**Client used:** Shoukaku

Output of `java -version`:
```
openjdk version "18.0.2.1" 2022-08-18
OpenJDK Runtime Environment Temurin-18.0.2.1+1 (build 18.0.2.1+1)
OpenJDK 64-Bit Server VM Temurin-18.0.2.1+1 (build 18.0.2.1+1, mixed mode, sharing)
```

Output of `java -jar Lavalink.jar --version`:
```
Version:        3.7.2
Build time:     21.01.2023 17:48:55 UTC
Branch          HEAD
Commit:         338497c
Commit time:    21.01.2023 17:46:26 UTC
JVM:            18.0.2.1
Lavaplayer      1.3.99.2-original
```

**Comment by topi314** [comment, marked as answer]
to connect to something running inside a docker container refer to dockers networking docs
https://docs.docker.com/network/
